packages/react-app/src/types.d.ts

```diff
@@ -1,4 +1,3 @@
 // TODO :  Declare full library type
 declare module '@1hive/1hive-ui';
 declare module 'use-wallet';
```

Comment:
Both now have types definitions.

1hive-ui in the last version 

And use-wallet need be check if break everything if we get it updated.

Reply:
So are we removing these 2 lines ?

Reply:
We will do it In another refactor

Reply:
#227 nable 1hive-ui type definition